Code/GraphMol/FileParsers/testMolSupplier.cpp

```diff
@@ -104,210 +105,218 @@ int testMolSup() {
-      TEST_ASSERT(nmol);
-      {
-        Bond *bnd = nmol->getBondWithIdx(1);
-        TEST_ASSERT(bnd);
-        TEST_ASSERT(bnd->getStereoAtoms() == INT_VECT({0, 3}));
-        TEST_ASSERT(bnd->getStereo() == Bond::STEREOTRANS);
-      }
-      {
-        Bond *bnd = nmol->getBondWithIdx(3);
-        TEST_ASSERT(bnd);
-        TEST_ASSERT(bnd->getStereoAtoms() == INT_VECT({2, 5}));
-        TEST_ASSERT(bnd->getStereo() == Bond::STEREOCIS);
+        // All atoms have the string prop
+        TEST_ASSERT(atom->hasProp("s_m_dummy"));
+        TEST_ASSERT(atom->getProp<std::string>("s_m_dummy") ==
+                    std::to_string(19 - i));
       }
+
+      TEST_ASSERT(maesup.atEnd());
     }
-    {  // Chiralities (these get CIP codes)
-      std::unique_ptr<ROMol> nmol(maesup.next());
-      TEST_ASSERT(nmol);
-      {
-        Atom *at = nmol->getAtomWithIdx(1);
-        TEST_ASSERT(at);
-        TEST_ASSERT(at->getChiralTag() == Atom::CHI_TETRAHEDRAL_CCW);
-        TEST_ASSERT(at->getProp<std::string>(common_properties::_CIPCode) ==
-                    "R");
-      }
-      {
-        Atom *at = nmol->getAtomWithIdx(3);
-        TEST_ASSERT(at);
-        TEST_ASSERT(at->getChiralTag() == Atom::CHI_TETRAHEDRAL_CCW);
-        TEST_ASSERT(at->getProp<std::string>(common_properties::_CIPCode) ==
-                    "S");
+
+    {  // Test parsing stereo properties. Mol is 2D and has stereo labels.
+      fname = rdbase + "/Code/GraphMol/FileParsers/test_data/stereochem.mae";
+      MaeMolSupplier maesup(fname);
+
+      {  // Stereo bonds. These get overwritten by the double bond detection.
+        std::unique_ptr<ROMol> nmol(maesup.next());
+        TEST_ASSERT(nmol);
+        {
+          Bond *bnd = nmol->getBondWithIdx(1);
+          TEST_ASSERT(bnd);
+          TEST_ASSERT(bnd->getStereoAtoms() == INT_VECT({0, 3}));
+          TEST_ASSERT(bnd->getStereo() == Bond::STEREOTRANS);
+        }
+        {
+          Bond *bnd = nmol->getBondWithIdx(3);
+          TEST_ASSERT(bnd);
+          TEST_ASSERT(bnd->getStereoAtoms() == INT_VECT({2, 5}));
+          TEST_ASSERT(bnd->getStereo() == Bond::STEREOCIS);
+        }
       }
-    }
-    {  // Pseudochiralities (no CIP codes)
-      std::unique_ptr<ROMol> nmol(maesup.next());
-      TEST_ASSERT(nmol);
-      {
-        Atom *at = nmol->getAtomWithIdx(2);
-        TEST_ASSERT(at);
-        TEST_ASSERT(at->getChiralTag() == Atom::CHI_TETRAHEDRAL_CW);
-        TEST_ASSERT(!at->hasProp(common_properties::_CIPCode));
+      {  // Chiralities (these get CIP codes)
+        std::unique_ptr<ROMol> nmol(maesup.next());
+        TEST_ASSERT(nmol);
+        {
+          Atom *at = nmol->getAtomWithIdx(1);
+          TEST_ASSERT(at);
+          TEST_ASSERT(at->getChiralTag() == Atom::CHI_TETRAHEDRAL_CCW);
+          if (useLegacy) {
+            TEST_ASSERT(at->getProp<std::string>(common_properties::_CIPCode) ==
+                        "R");
+          }
+        }
```

Comment:
should we add an else?

Reply:
no, there are no CIP codes set by default in those cases